index.html

```diff
@@ -1,199 +1,277 @@
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+        <button type="submit" class="about-me-btn">Get My Resume</button>
       </div>
-      <div class="framework-card about-box">
-        <img src="images/Icon2.png" alt="icon for skills">
-        <h4 class="lang-title">Frameworks</h4>
-        <ul class="about-me-tags">
-          <li class="tag-about-items">Bootstrap</li>
-          <li class="tag-about-items">Ruby on rails</li>
-          <li class="tag-about-items">RSpec</li>
-          <li class="tag-about-items">Capybara</li>
-          <li class="tag-about-items">Salenium</li>
-        </ul>
+      <div class="grid-container">
+        <div class="lang-card about-box">
+          <img src="images/Icons.png" alt="icon languages" />
+          <h4 class="lang-title">Languages</h4>
+          <ul class="about-me-tags">
+            <li class="tag-about-items">JavaScript</li>
+            <li class="tag-about-items">Ruby</li>
+            <li class="tag-about-items">HTML</li>
+            <li class="tag-about-items">CSS</li>
+          </ul>
+        </div>
+        <div class="framework-card about-box">
+          <img src="images/Icon2.png" alt="icon for skills" />
+          <h4 class="lang-title">Frameworks</h4>
+          <ul class="about-me-tags">
+            <li class="tag-about-items">Bootstrap</li>
+            <li class="tag-about-items">Ruby on rails</li>
+            <li class="tag-about-items">RSpec</li>
+            <li class="tag-about-items">Capybara</li>
+            <li class="tag-about-items">Salenium</li>
+          </ul>
+        </div>
+        <div class="skills-card about-box">
+          <img src="images/icon-skills.png" alt="icon for skills" />
+          <h4 class="lang-title">Skils</h4>
+          <ul class="about-me-tags">
+            <li class="tag-about-items">codeKit</li>
+            <li class="tag-about-items">GitHub</li>
+            <li class="tag-about-items">codepen</li>
+            <li class="tag-about-items">Gitlap</li>
+            <li class="tag-about-items">Terminal</li>
+          </ul>
+        </div>
       </div>
-      <div class="skills-card about-box">
-        <img src="images/icon-skills.png" alt="icon for skills">
-        <h4 class="lang-title">Skils</h4>
-        <ul class="about-me-tags">
-          <li class="tag-about-items">codeKit</li>
-          <li class="tag-about-items">GitHub</li>
-          <li class="tag-about-items">codepen</li>
-          <li class="tag-about-items">Gitlap</li>
-          <li class="tag-about-items">Terminal</li>
+    </section>
+
+    <section class="form-section">
+      <div class="form-container">
+        <h2 class="form-title">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/MohamedHNoor" method="post">
```

Comment:
- [ ] First of all, you got an error of  `element “form”: Illegal character in path segment: “{” is not allowed.` from webhint in the previous commit because the sign `{` is dangerous when it comes to posting data, however, the {form-id} is just a placeholder so you know that you need to put your id in there, now you added `MohamedHNoor` which is not a correct id generated by `formspree` the form-id should be like this format: `xzb34dvd`  ( please do not use it , it's just an example) 


- **_How to get your own id_**: 

1. you need to create an account in [formspree](https://formspree.io/register)
2. you create a new form by clicking on this button 


![image](https://user-images.githubusercontent.com/89282221/186213410-bee588c7-e316-4570-8e66-13be7021d2b7.png)

4.  that's it, after you create the form you will get your own id that you can use inside the post method in your form 
![image](https://user-images.githubusercontent.com/89282221/186213709-680249ff-b2c7-4691-aa65-0ea912fb0d30.png)